include/atalk/cast.h

```diff
@@ -71,36 +71,16 @@ extern "C" {
 #define CAST_BLOCK	8
 #define CAST_KEY_LENGTH	16
 
-typedef struct cast_key_st
-	{
+typedef struct cast_key_st {
 	CAST_LONG data[32];
-	} CAST_KEY;
+} CAST_KEY;
 
-#ifndef NOPROTO
```

Comment:
What’s the reason for this local modification?

Reply:
It's an irrelevant conditional